design-documents/graph-ql/coverage/available-stores.md

```diff
@@ -0,0 +1,73 @@
+# Get available stores for website
+
+### Use case:
+Implementing a store switcher; it is necessary to know which stores are available and some basic info about them (i.e. store code)
+
+### Implementation detail:
+Based on the store code passed via header (or default), returns the storeConfig for all stores available under the same website.
+
+### Proposed schema
+```graphql
+type Query {
+    availableStores: [StoreConfig] @doc(description: "Get a list of available stores")
```

Comment:
Do we want to name this something like `availableStoreViews`? For UI developers in a headless world, the GraphQL schema will often be their first introduction to some Magento concepts.

Referencing "store view" instead of just "store" will make it much easier to Google for the right concept (test googling `magento stores` vs `magento store views`)

Thoughts?

Reply:
Often times in the frontend we refer to store views as stores interchangeably. I think using just "store" is more consistent; e.g. StoreConfig and the Store header we use for store code. it's a good point. maybe just changing the description will be enough?

Reply:
> maybe just changing the description will be enough?

The one downside there is that most tools like GraphiQL don't cover the `description` in their search field, so searching for something like `store view` wouldn't net any results.

BUT, you do raise a good point re: consistency vs accuracy. My feelings here aren't too strong, so I'll leave this as a judgement call for you to make 👍